A/armadillo/build_tarballs.jl

```diff
@@ -20,31 +20,11 @@ FLAGS=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
        -DCMAKE_BUILD_TYPE=Release
        -DBUILD_SHARED_LIBS=ON)
 
-if [[ "${nbits}" == 64 ]] && [[ "${target}" != aarch64* ]]; then
-    FLAGS+=(-Dopenblas_LIBRARY="${libdir}/libopenblas64_.${dlext}")
-    # Force Armadillo's CMake configuration to accept OpenBLAS as a LAPACK
-    # replacement.
-    FLAGS+=(-DLAPACK_LIBRARY="${libdir}/libopenblas64_.${dlext}")
+if [[ "${nbits}" == 64 ]]; then
 
-    SYMB_DEFS=()
-    for sym in sasum dasum snrm2 dnrm2 sdot ddot sgemv dgemv cgemv zgemv sgemm dgemm cgemm zgemm ssyrk dsyrk cherk zherk; do
-        SYMB_DEFS+=("-D${sym}=${sym}_64")
-    done
     if [[ "${target}" == *-apple-* ]] || [[ "${target}" == *-mingw* ]]; then
-        FLAGS+=(-DALLOW_OPENBLAS_MACOS=ON)
+        FLAGS+=(-DALLOW_BLAS_LAPACK_MACOS=ON)
```

Comment:
I don't think this needs to change either.  From the Armadillo README:

```
    - On macOS, to enable the detection of OpenBLAS,
      use the additional ALLOW_OPENBLAS_MACOS option when running cmake:

      cmake -DALLOW_OPENBLAS_MACOS=ON .

      Note: depending on your installation, OpenBLAS may masquerade as standard BLAS.
      To detect standard BLAS and LAPACK, use the ALLOW_BLAS_LAPACK_MACOS option:

      cmake -DALLOW_BLAS_LAPACK_MACOS=ON .
```

Unless OpenBLAS is now distributed differently, I suspect that this change is not needed.